mangaki/mangaki/utils/anidb.py

```diff
@@ -132,6 +132,34 @@ def _build_work_titles(self,
 
         return missing_titles
 
+    def get_XML(self, anidb_aid):
```

Comment:
In Python, there is [a convention for the naming style](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles) (which you're not required to entirely follow), but in this example, `get_XML` style is different from the usual convention in the code, `get_xml` is more appropriate, if we used `somethingSomething` as naming convention, then `getXML` would have been acceptable I think.